### PR TITLE
darkmode retains after refresh

### LINKED ISSUE #221: Dark mode gets switched off after refreshing the page
#### I am submitting a -
* [] **Feature Request**
* [x] **Bug Report**

#### Describe your bug/feature -
Dark mode gets switched off after refreshing the page rather it should retain its value after refreshing.

#### Current behaviour -
Dark mode gets switched off after refreshing the page.

#### Expected behavior -
It should retain its previous value after refreshing. 
#### Are you working on this issue? (Yes/No)
yes

### PR DESCRIPTION
**Pull request by animesh-007**
Fixes #221

### CODE REVIEW

**Comment by animesh-007** [comment]
> image
> do not access localstorage values using dot

Ok Now I used getItem() but I m unable to keep slider ON on refresh.
Can you suggest anything?

**Comment by animesh-007** [comment]
> The dark mode retains, but the button state is not the same. check screenshot.
> image

Now it is resolved

**Comment by animesh-007** [comment]
I checked on my localhost and on netlify preview too but It's working fine over there.
Screen Shot 2020-05-17 at 10 09 00 PM
The above screenshot is taken after ctrl+shift+r.
Still, I m looking to find if any problem persists.
Can you suggest anything?

File: docs/index.html [hunk 1]
```diff
@@ -25,22 +25,27 @@
     	var url2 = "//unpkg.com/docsify/themes/vue.css";
     	var stylesheet = document.getElementById("stylesheet");
     	var show = false;
-
+		var darkmode = localStorage.getItem('darkmode')
```
Comment (YashKumarVerma) [inline docs/index.html:28]: can you align it properly, also similar other cases in this file ?

rest LGTM

**Comment by animesh-007** [comment]
> @animesh-007 Can you solve the merge conflicts?

@YashKumarVerma I solved merge conflicts.

**Comment by YashKumarVerma** [comment]
@ShubhankarKG @schezfaz @eKuG @coding-CEO @Shivansh2407 
Can you check if this is working fine.

**Comment by schezfaz** [comment]
Checked the following:
- [x] Toggle to dark mode --> Refresh page --> Dark more persists 
- [x] Toggle to dark mode on home page --> navigate to another page (say Video demos) --> hit refresh --> Dark mode persists
- [x] Keep pages in light mode --> Refresh page --> Light mode persists
- [x] Whether toggle button state persists on refresh

All cases seem to work well, looks good to me!

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions @aayush-05 , @schezfaz  for quick review.

**Pull request merged**